cwms-http-client/src/main/java/mil/army/usace/hec/cwms/http/client/HttpRequestBuilderImpl.java

```diff
@@ -57,10 +66,11 @@ public HttpRequestBuilderImpl(ApiConnectionInfo apiConnectionInfo, String endpoi
         }
         this.httpUrl = url;
         this.endpoint = Objects.requireNonNull(endpoint, "Cannot process request against the API root endpoint");
+        method = HttpRequestMethod.GET; //default method
```

Comment:
Why do we need a default? Doesn't the fluent API ensure that a method is set?

Reply:
Yeah that's correct. Forgot to remove the default set.